src/System.CommandLine.DragonFruit.Tests/ConfigureFromMethodTests.cs

```diff
@@ -220,7 +220,7 @@ public void Options_are_not_built_for_infrastructure_types_exposed_by_method_par
             var options = handlerMethod.BuildOptions();
 
             options.Should()
-                   .NotContain(o => o.Argument.ArgumentType == type);
+                   .NotContain(o => o.ArgumentType == type);
```

Comment:
I wonder if `ValueType` would be a better name for this property. That would match the `IValueDescriptor` interface, for what it's worth.